Gemfile

```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gem "fastlane"
+gem "cocoapods"
```

Comment:
I would try to specify a version in case in x times the library is not compatible with these gem in dev you can use
```ruby
gem 'fastlane', '~>2.14.2'
gem 'cocoapods', '~>1.2.0'
```

see http://bundler.io/v1.12/gemfile.html for version specifier